recipe/meta.yaml

```diff
@@ -36,12 +36,16 @@ requirements:
     - python
     - conda-env >=2.6
     - enum34                 # [py<34]
-    - futures                # [py<34]
+    - futures >=3.0.0        # [py<34]
     - menuinst               # [win]
     - pycosat >=0.6.3
     - pyopenssl >=16.2.0
     - requests >=2.12.4,<3
     - ruamel_yaml >=0.11.14
+  run_constrained:
+    - conda-build >=2.1
+    - cytoolz >=0.8.1
+    - setuptools >=36.4
```

Comment:
If nothing else changed, `- setuptools >=31.0.1` should suffice, see https://github.com/conda/conda/pull/6654#issuecomment-355688452.

Reply:
Should we worry about this? Are there packages actually pinning `setuptools`?

Reply:
The setuptools requirement here is to force an upgrade that fixes a bug, as I guess discussed in that issue. I feel like I bumped it again to something even more aggressive and recent for a reason, but again don’t remember exactly why.

Reply:
`git blame` says it hasn't been touched after https://github.com/conda/conda/pull/6654, so we can/should use `- setuptools >=31.0.1` here.

Reply:
My feeling is we should follow `conda`'s version constraints here. If they change in `conda`, happy to mirror them here.

Reply:
Seems fair, I'll adjust them upstream.